subprojects/docs/src/samples/announce/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+// tag::use-plugin[]
+plugins {
+    announce
+}
+// end::use-plugin[]
+
+// tag::announce-plugin-conf[]
+announce {
+    username = "myId"
+    password = "myPassword"
+}
+// end::announce-plugin-conf[]
+
+// tag::announce-usage[]
+val helloWorld by tasks.creating {
```

Comment:
I think we want to avoid use of the delegated properties syntax here. Right @eskatos?

Reply:
I think we want

```
task("helloWorld") {
    doLast {
        println("Hello, world!")
    }
}

tasks.getByName("helloWorld").doLast {
    announce.announce("helloWorld completed!", "twitter")
    announce.announce("helloWorld completed!", "local")
}
```

Reply:
The idea was to avoid using a string to identify the task, since it's created and then modified. But I can have that without using a delegated property, using

```
val helloworld = task("helloWorld") {
    doLast {
        println("Hello, world!")
    }
}

helloworld.doLast {
    announce.announce("helloWorld completed!", "twitter")
    announce.announce("helloWorld completed!", "local")
}
```

or 

```
val helloworld = tasks.create("helloWorld") {
    doLast {
        println("Hello, world!")
    }
}

helloworld.doLast {
    announce.announce("helloWorld completed!", "twitter")
    announce.announce("helloWorld completed!", "local")
}
```

which, IMHO, is cleaner since it better shows that we're creating a new task, and it doesn't use an Incubating method. 

What do you think @eriwen @eskatos ?
Out of curiosity, why do we want to avoid delegated properties? Is it because it's considered too advanced? Or unstable? Or simply to be closer to the groovy code?

Reply:
I personally like delegated properties for type-safety, but the most important thing is that these samples are:
 * Near exactly equivalent to the Groovy samples
 * Understood without Kotlin-specific syntax idioms (e.g. delegated properties)

That doesn't mean we always forego delegated properties, but for the user manual we should as a general rule. 

Does that answer your question?

Reply:
Yes, thank you @eriwen